aten/src/ATen/native/cpu/GridSamplerKernel.cpp

```diff
@@ -764,6 +778,197 @@ struct ApplyGridSample<scalar_t, 2, GridSamplerInterpolation::Nearest,
   }
 };
 
+template<typename scalar_t, GridSamplerPadding padding, bool align_corners>
```

Comment:
Comment here explaining the approach. This is actually using the convolutional approach with alphea = -.75, right?

Reply:
What motivated the choice of -.75 for alpha, by the way?

Reply:
There is a [discussion](https://github.com/pytorch/pytorch/pull/9849#discussion_r234878156) in UpSampleBicubic. It seems PIL use -0.5 and also in wiki page. But... I just choose the same value as UpSampleBicubic

Reply:
I remember why the alpha should be -.75. I use the `get_cubic_upsample_coefficients` in `aten/src/ATen/native/cuda/UpSample.cuh` and `aten/src/ATen/native/UpSample.h`. Hence the value should be the same in this file. There is another small issue, since the backward calculation also need alpha. I hard code the value in  `get_cubic_coefficients_grad`. There would be a problem if someone in the future want to modify this value.

Reply:
That makes a lot of sense. Would you add a brief comment explaining that in the code, too?

Reply:
Is that clear to the write the full reason in the code? There are three sections(cpu, kernel, cuda) about the cubic interpolation. I'll prefer to write it in the python document, for example

Note: ``mode='bicubic'`` is implemented by [cubic convolution algorithm](https://en.wikipedia.org/wiki/Bicubic_interpolation). The constant `\alpha` is chosen to be -0.75. The value might be different from other image processing package, for example PIL use `\alpha=-0.5` and opencv use `alpha=-0.75`. Besides, bicubic interpolation is possible to cause overshoot, in other words it can produce negative values or values greater than 255 for images. Explicitly call ``result.clamp(min=0, max=255)`` if you want to reduce the overshoot when displaying the image.

Reply:
I understand. Elaborating in the documentation about the algorithm and choice of alpha value would be a good idea.

Reply:
I like the note you added. I made a suggestion (below) to make it a little more precise. Let me know what you think.

Reply:
That looks great. It is more clear to explicitly show the range. I will update the document as soon as possible. Thanks